public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -271,6 +274,8 @@ export function initDashboard(args: InitDashboardArgs): ThunkResult<void> {
       })
     );
 
+    trackDashboardLoaded(dashboard, versionBeforeMigration);
```

Comment:
FYI the lines above already publishes a dashboard loaded on the event bus which is picked up by quite a few datasources already (and they emit a RS event as a result).

Not a huge deal breaker but that means that this PR would now have 2 RS event for one dashboard loaded 😉 . Could be good to clean up at some point?

Reply:
Hey @yaelleC,
Thanks for the heads up. If a dashboard uses various datasources that uses the dashboard loaded event then this was already the case. I'm not sure how to clean this up. What do you suggest?

Reply:
Hey, I did not mean to block the PR with this comment - but rather to make sure @grafana/plugins-platform are kept in the loop so our user events are kept in line and sanity checked.

The only solution I see to avoid the duplicate is use this PR's RS event only and fill it with DS info, but that means adding an endpoint to the datasources (to get DS specific info aggregated) 🤷‍♀️

Reply:
I am not sure if I follow, why don't you extend `DashboardLoadedEvent` with the new info you want? (theme, schemaVersion, etc...)

I don't see any DS info being used?

Reply:
DashboardLoaded is mostly used by data sources. I'm not sure if variable types makes sense for data sources. I kind of wanted to separate it. I'm up to what ever @grafana/dashboards-squad decide on.

Reply:
ah, okay, now I get it, it's the data source the one that `reportInteraction` in the existing code and in this PR you are reporting an interaction per dashboard. 

In that case I think it's fine to have two different objects to report (to avoid to send the whole `queries`, which is part of `DashboardLoadedEvent`).